.github/workflows/e2e-arm64-nightly.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
With addition of ARM nightly per branch I think we are getting the limit of clarity of `./github/workflows` directory.
Maybe we could consider creating subdirs and grouping workflows by purpose:
* `./github/workflows/periodics`
* `./github/workflows/presubmits`

Leaving the a idea for a followup.